### PR TITLE
Updated homepage and added the project status link

### PR DESCRIPTION
**Pull request by natashapl**
### This pull request includes the following update:

- Updated The Notify.gov brow on the homepage
- Added the project status link to the top nav and changed the style to match @jonathanbobel 's design.

[PR Preview](https://federalist-37831b52-7633-429a-9a79-14298034dd54.sites.pages.cloud.gov/preview/gsa-tts/10x-website/add-project-status-link-to-menu/)

### CODE REVIEW

File: _includes/components/header.html [hunk 1]
```diff
@@ -55,7 +55,109 @@
...
+      <a href="{{ '/' | url }}">
+        <span id="basic-logo">
+          <img alt="10 x home" width="72" height="36" src="{{ '/assets/images/10x-logo.webp' | url }}">
+        </span>
+      </a>
+    </div>
+    <button type="button" class="usa-menu-btn">Menu</button>
+  </div>
+  <nav aria-label="Primary navigation" class="usa-nav">
+    <div class="usa-nav__inner">
```
Comment (jonathanbobel) [inline _includes/components/header.html:71]: I was messing around with the z-index here and bumping it up allowed for the tabbed focus state to be visible over the gradient background below. That might not be the best solution, so I'd experiment in the code.

Comment (jonathanbobel) [reply _includes/components/header.html:71]: (specifically "usa-nav__inner")

Comment (natashapl) [reply _includes/components/header.html:71]: Hi @jonathanbobel . The branch has been updated. Please review.

**Comment by natashapl** [comment]
Hi @iamjolly . Hope all is well. In this task, we've updated the main top navigation of the 10x website and added a "Project Status" link. When you have a moment, could you please perform some additional accessibility testing and let us know if we overlooked anything? Thanks so much!

[Click here to access the branch.](https://federalist-37831b52-7633-429a-9a79-14298034dd54.sites.pages.cloud.gov/preview/gsa-tts/10x-website/add-project-status-link-to-menu/)

**Comment by iamjolly** [comment]
@natashapl - Things check out fine in general on the new link, with a couple of minor comments/suggestions:

1. The Github icon is cool, but the accessible name for the link is simply "Project Status". I think as-is, it is OK, but would be even better to be "Project Status on Github" as the accessible name for the link. For this, you may want to use visually-hidden text in a `span` within the `a` to add to the link text. It's optional, but seems like it could be a nice touch.
2. Focus outline is great for this link and is fine as-is. I really like the `:hover` color change from white to light blue and would recommend that also be applied to the `:focus` in the CSS, along with the focus outline. Again, not a big deal if it's not implemented, just a nice touch visually.

**Comment by natashapl** [comment]
Hi @iamjolly . Thanks for you suggestions. They've both been implemented. Please review when you get a chance. By the way, I added you as a reviewer. Thanks.

**Review by iamjolly — APPROVED** [review]
these look good to me - thanks for working through this!

**Comment by jonadecker** [comment]
This looks really good @natashapl! Thanks for your help on this @iamjolly

**Review by jonathanbobel — COMMENTED** [review]
LGTM

**Pull request merged**